website/pages/404.jsx

```diff
@@ -15,18 +18,42 @@ export default function NotFound() {
       })
   }, [])
 
-  return (
-    <div id="p-404" className="g-grid-container">
-      <h1 className="g-type-display-1">Page Not Found</h1>
+  const defaultMessage = (
+    <>
       <p>
-        We&lsquo;re sorry but we can&lsquo;t find the page you&lsquo;re looking
+        We&rsquo;re sorry but we can&rsquo;t find the page you&rsquo;re looking
         for.
       </p>
       <p>
         <Link href="/">
           <a>Back to Home</a>
         </Link>
       </p>
+    </>
+  )
+
+  const reg = /\/(?<version>v\d+[.]\d+[.](\d+|x))/g
+  const matches = reg.exec(asPath)
+  const docsMessage = (
+    <>
+      <p>
+        We&rsquo;re sorry, but this page does not exist for version&nbsp;
+        <b>{matches?.groups?.version}</b>.
+      </p>
+      <p>
+        Try viewing the&nbsp;
+        <Link href={asPath.replace(reg, '')}>
```

Comment:
@thiskevinwang This is just showing up as `/` 🤔 : https://waypoint-git-kevin-404-for-docs-hashicorp.vercel.app/plugins/v0.4.x/helm

Reply:
Weird. Clicking it still redirects to `/plugins/helm` for me.
Looking...